xicam/gui/windows/mainwindow.py

```diff
@@ -169,7 +169,8 @@ def open(self, header):
         elif isinstance(header, NonDBHeader):
             self.currentGUIPlugin.appendHeader(header)
         else:
-            raise TypeError(f"Cannot open {header}.")
+            self.currentGUIPlugin.appendCatalog(header)
```

Comment:
What type isn't being caught here that you are trying to open in a gui plugin?

Reply:
This commit is not meant for merging, at least not yet anyway. This was so that Ron and you had a chance to see the changes in progress. 

In regards to the question, this needs to be replaced by a Catalog at some point. The Globus plugin currently emitting a Filepath that is neither a header or Catalog, but this will change soon.